src/components/Explore/Explore.tsx

```diff
@@ -116,17 +117,23 @@ const Explore: React.FunctionComponent<{
 
   const currentMetricName = getMetricName(currentMetric);
 
-  const currentLocation = useMemo(() => findLocationForFips(fips), [fips]);
-  const autocompleteLocations = useMemo(() => getAutocompleteLocations(fips), [
-    fips,
+  const currentLocations = useMemo(() => fipsList.map(findLocationForFips), [
```

Comment:
i'm a little surprised this is helpful

Reply:
Good point, is probably not useful in this case, `findLocationForFips` is not an expensive computation, we don't need `useMemo` here.